app/service.js

```diff
@@ -1,13 +1,21 @@
 const bodyParser = require('body-parser');
 const express = require('express');
 const domino = require('domino');
-const fetch = require('node-fetch');
 const urlparse = require('url');
 const {getMetadata} = require('page-metadata-parser');
+require('isomorphic-fetch');
+
+const errorMessages = {
+  headerRequired: 'The content-type header must be set to application/json.',
+  urlsRequired: 'The post body must be a JSON payload in the following format: {urls: ["http://example.com"]}.',
+  maxUrls: 'A maximum of 20 urls can be sent for processing in one call.'
```

Comment:
This is the only instance of `maxUrls` I see. Not sure if we need code/tests for this as well, or leaving for a future PR.